src/_pytest/config/__init__.py

```diff
@@ -172,7 +172,10 @@ def directory_arg(path, optname):
 builtin_plugins.add("pytester")
 
 
-def get_config(args=None, plugins=None):
+def get_config(
+    args: Optional[Union[List, Tuple]] = None,
```

Comment:
`List` and `Tuple` without a `[]` is equivalent to `List[Any]` and `Tuple[Any]`. Are there narrower types that can be used?

If yes, it's OK to leave them out for now, leaving them out indicates a TODO.

If no, I prefer to put explicit `List[Any]`, this it's clear it can really be anything and is not just omitted/TODO.

Reply:
Also changing to `args: Optional[Sequence[Union[str, py.path.local]]] = None,`.

Reply:
JFI: postponing this for now, not trivial it seems (also the one above, will delete the comments).